### PR TITLE
exec.cc: Free malloced pointer after use in scan_exec_arg()

### PR DESCRIPTION
**Pull request by lasers**
Addresses https://github.com/brndnmtthws/conky/issues/605

### CODE REVIEW

**Comment by lasers** [comment]
@su8 FYI. I don't see your contributions being counted when I used your commits, but certainly fewer duplicate PRs. Do whatever you want.

File: src/exec.cc [hunk 1]
```diff
@@ -286,6 +288,9 @@ void scan_exec_arg(struct text_object *obj, const char *arg,
     ed->cmd = strndup(cmd != nullptr ? cmd : "", text_buffer_size.get(*state));
     obj->data.opaque = ed;
   }
+  if (cmd != nullptr && 1 == free_cmd) {
+    free((char *)cmd);
```
[the remark this answers is not in the document]
Comment (su8) [reply src/exec.cc]: Go away robot :laughing:

[the remark this answers is not in the document]
Comment (lasers) [reply src/exec.cc]: @su8 Wait for the owner to bypass this?

[the remark this answers is not in the document]
Comment (su8) [reply src/exec.cc]: @lasers Yes master.

**Pull request merged**